Description: Avoid installing a non-free files
 * GenericCMYK triggers the "license-problem-md5sum-non-free-file
   scribus/profiles/GenericCMYK.icm usual name is Fogra27L.icm." lintian error.
Author: Mattia Rizzolo <mattia@mapreri.org>
Author: Oleksandr Moskalenko <malex@debian.org>
Bug-Debian: https://bugs.debian.org/741666
Forwarded: not-needed
Last-Update: 2014-07-27

--- a/resources/editorconfig/CMakeLists.txt
+++ b/resources/editorconfig/CMakeLists.txt
@@ -12,7 +12,6 @@
 300_lilypond.xml
 lilypond.png
 400_dot.xml
-dot.svg
 500_povray.xml
 povray_32.png
 
--- a/scribus/profiles/CMakeLists.txt
+++ b/scribus/profiles/CMakeLists.txt
@@ -3,10 +3,7 @@
 )
 
 INSTALL(FILES
-GenericCMYK.icm
 GenericCMYK.txt
 Readme
-sRGB.icm
-srgb.license
   DESTINATION ${SHAREDIR}profiles
 )
--- a/scribus/CMakeLists.txt
+++ b/scribus/CMakeLists.txt
@@ -2,7 +2,6 @@
   ADD_DEFINITIONS(-DQT_DLL)
 ENDIF(WIN32)
 
-ADD_SUBDIRECTORY(doc)
 ADD_SUBDIRECTORY(dtd)
 ADD_SUBDIRECTORY(manpages)
 ADD_SUBDIRECTORY(colormgmt)
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -804,7 +804,6 @@
 
 #Add our source subdirs
 ADD_SUBDIRECTORY(scribus)
-ADD_SUBDIRECTORY(resources/dicts)
 ADD_SUBDIRECTORY(resources/editorconfig)
 ADD_SUBDIRECTORY(resources/icons)
 ADD_SUBDIRECTORY(resources/keysets)
